app/keepers/modules.go

```diff
@@ -0,0 +1,84 @@
+package keepers
```

Comment:
What's the motivation for moving this to a dedicated keepers package instead of `app`?

Reply:
The goal is to get more of the keeper/module scaffolding outside of the app package, so its importable by other packages. Maybe this doesn't make sense though.

The `AppKeepers` struct was needed in its own package, to fix API complexity problems in our upgrades registration code. (We just want to give it access to all keepers it can modify, while having each upgrade be its own package)